clang/test/Driver/cheri/freebsd-cheri-libcheri.c

```diff
@@ -0,0 +1,41 @@
+// RUN: %plain_clang_cheri_triple_allowed -no-canonical-prefixes \
+// RUN:   -target cheri-pc-freebsd11 %s                              \
```

Comment:
```suggestion
// RUN:   -target mips64-pc-freebsd11 -cheri=128 %s                              \
```
Maybe avoid the deprecated "cheri-" triple?